crates/environ/src/builtin.rs

```diff
@@ -163,7 +163,14 @@ macro_rules! foreach_builtin_function {
             tc_print_pointer(vmctx: vmctx, arg : pointer);
 
             // Returns an index for Wasm's `table.grow` instruction for `contobj`s.
-            table_grow_cont_obj(vmctx: vmctx, table: i32, delta: i32, init: pointer) -> i32;
+            // Note that the initial Option<VMContObj> (i.e., the value to fill
+            // the new slots with) is split into two arguments: The underlying
+            // continuation reference and the revision count.
+            // If `unsafe_disable_continuation_linearity_check` is enabled, the revision value is arbitrary.
+            // To denote the continuation being `None`, `init_contref` may be 0.
+            table_grow_cont_obj(vmctx: vmctx, table: i32, delta: i32, init_contref: pointer, init_revision : i64) -> i32;
```

Comment:
I guess it would be nice to extend the interface of libcalls API to support 128 bit wide values.

Reply:
I am not suggesting doing it now. But I think it will be simpler than you think. We should be able to map a hypothetical `i128` to Rust `u128` just as `i32` maps to `u32`, etc.